_posts/news/2018-10-22-announcing-scalajs-1.0.0-M6.md

```diff
@@ -0,0 +1,480 @@
+---
+layout: post
+title: Announcing Scala.js 1.0.0-M6
+category: news
+tags: [releases]
+permalink: /news/2018/10/22/announcing-scalajs-1.0.0-M6/
+---
+
+
+We are very excited to announce the release of Scala.js 1.0.0-M6!
+
+This development release is mostly intended for testing purposes, and as a synchronization point with library authors so that they can start upgrading in preparation for the final release.
+
+As the change in "major" version number witnesses, this release is *not* binary compatible with 0.6.x, nor with the previous milestones of the 1.x series.
+Libraries need to be recompiled and republished using this milestone to be compatible.
+
+Moreover, this release is not entirely source compatible with 0.6.x either.
+We expect, however, that further milestones for 1.0.0 will stay backward source compatible with this milestone.
+
+These release notes contain cumulative changes with respect to 0.6.25.
+Compared to 1.0.0-M5, the following changes are noteworthy:
+
+* Scala.js has been relicensed under the Apache License 2.0, following the corresponding relicensing of Scala upstream
+* Add support for Scala 2.12.7 and 2.13.0-M5
+* Overall bring up-to-date with improvements and bug fixes available in 0.6.25
+* The `scalajs-stubs` artifact has been moved to [its own repository](https://github.com/scala-js/scala-js-stubs), with its own versioning scheme; its latest release is 1.0.0-RC1
```

Comment:
Should we mention:
- Removal of `TestUtils` in favor of Reflect (breaking for testing frameworks).
- Introduction of `org.scalajs.testing` (breaking for testing integrators).
- `withSupportsExit` for JSEnv test suite (breaking for `JSEnv` implementors).